packages/cli/src/databases/migrations/mysqldb/1681134145996-AddUserActivatedProperty.ts

```diff
@@ -0,0 +1,59 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { getTablePrefix, logMigrationEnd, logMigrationStart } from '@db/utils/migrationHelpers';
+import { userSettings } from '@/Interfaces';
+
+export class AddUserActivatedProperty1681134145996 implements MigrationInterface {
+	name = 'AddUserActivatedProperty1681134145996';
+
+	async up(queryRunner: QueryRunner): Promise<void> {
+		logMigrationStart(this.name);
+
+		const tablePrefix = getTablePrefix();
+
+		const activatedUsers: userSettings[] = await queryRunner.query(
+			`SELECT DISTINCT sw.userId AS id,
+				JSON_SET(COALESCE(u.settings, '{}'), '$.userActivated', true) AS settings
+			FROM ${tablePrefix}workflow_statistics AS ws
+						JOIN ${tablePrefix}shared_workflow as sw
+							ON ws.workflowId = sw.workflowId
+						JOIN ${tablePrefix}role AS r
+							ON r.id = sw.roleId
+						JOIN ${tablePrefix}user AS u
+							ON u.id = sw.userId
+			WHERE ws.name = 'production_success'
+						AND r.name = 'owner'
+						AND r.scope = 'workflow'`,
+		);
+
+		const updatedUsers = activatedUsers.map((user) =>
+			queryRunner.query(
+				`UPDATE ${tablePrefix}user SET settings = '${JSON.stringify(user.settings)}' WHERE id = '${
+					user.id
+				}' `,
+			),
+		);
+
+		await Promise.all(updatedUsers);
+
+		if (!activatedUsers.length) {
+			await queryRunner.query(
+				`UPDATE ${tablePrefix}user SET settings = JSON_SET(COALESCE(settings, '{}'), '$.userActivated', false)`,
+			);
+		} else {
+			const activatedUserIds = activatedUsers.map((user) => `'${user.id}'`).join(',');
```

Comment:
```suggestion
			const activatedUserIds = activatedUsers.map((user) => user.id).join(',');
```

Reply:
This will make the query fail since I will be sending the following:

`NOT IN (21e0dc0e-df8a-11ed-b5ea-0242ac120002) instead of NO IN ('21e0dc0e-df8a-11ed-b5ea-0242ac120002')`

Reply:
since user ids are string, I think these need to be wrapped (as done already).